sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -1136,5 +1136,7 @@ def test_issue_8458():
     x, y = symbols('x y')
     p1 = Piecewise((0, x == 0), (sin(x), True))
     assert p1.simplify() == sin(x)
-    p2 = Piecewise((x+1, Eq(x, -1)), (4*x + (y-2)**4, Eq(x, 0) & Eq(x+y, 2)), (sin(x), True))
+    p2 = Piecewise((x, Eq(x, 0)), (4*x + (y-2)**4, Eq(x, 0) & Eq(x+y, 2)), (sin(x), True))
     assert p2.simplify() == sin(x)
+    p3 = Piecewise((x+1, Eq(x, -1)), (4*x + (y-2)**4, Eq(x, 0) & Eq(x+y, 2)), (sin(x), True))
+    assert p3.simplify() == Piecewise(0, Eq(x, -1), (sin(x), True)
```

Comment:
Something appears to be wrong with the test since the syntax is not right on this last line which should be
```python
p3.simplify() == Piecewise((0, Eq(x, -1)), (sin(x), True))
instead of
p3.simplify() == Piecewise(0, Eq(x, -1), (sin(x), True)
```
That the tests didn't complain is a concern. ping @asmeurer

Reply:
That is (unfortunately/fortunately) an old version of the code. More a Github/caching issue if you see this, so no worries @asmeurer that test did fail.